<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wsaoriginal</class>
 <widget class="QMainWindow" name="wsaoriginal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> [BETA] WSA Gui (WSA(Orginial) METHOD)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/wsa.png</normaloff>:/image/wsa.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="switchbutton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>361</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch to Only WSA + OpenGapps method</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Powershell security mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="powershell1">
       <property name="text">
        <string>To run powershell, lower the security level and set it to 
the default permission level after the processes are finished.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="powershell2">
       <property name="text">
        <string>Lower the security level to run powershell.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="powershell_none">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>1021</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Made by Winfried. Licensed under the GNU Affero General Public License v3.0.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDialogButtonBox" name="endbutton">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::NoButton</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>461</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Which OS would you prefer for WSA?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ubuntu">
       <property name="statusTip">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Ubuntu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="opensusetw">
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="text">
        <string>OpenSUSE Tumbleweed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="debian">
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="text">
        <string>Debian</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="wsa_none">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>481</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Requirements</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="virtual_machine_platform">
       <property name="text">
        <string>Enable Virtual Machine Platform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="download_wsa">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Download WSA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="download_gapps">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Download OpenGapps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wsa_windows_dev">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Enable WSA Developer Mode and Windows Developer Mode.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Optional</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="adb">
       <property name="text">
        <string>Download and install Android Platform Tools + adds path 
(adb command)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wsatools">
       <property name="text">
        <string>Download and install WSATools (apk alternative) 
(It offers better graphical interface for APK)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_wsa_msstore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>311</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>MS Store for %1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionMy_GitHub"/>
    <addaction name="actionProject_Page"/>
    <addaction name="actionGo_to_licence"/>
    <addaction name="actionDonate"/>
    <addaction name="actionAbout"/>
    <addaction name="actionApplication_About"/>
   </widget>
   <widget class="QMenu" name="menuDownload">
    <property name="title">
     <string>Downloads and Links</string>
    </property>
    <addaction name="actionMS_Store_for_powershell"/>
    <addaction name="actionMS_Store_for_WSL"/>
    <addaction name="actionMS_Store_for_WSA"/>
    <addaction name="actionMS_Store_for_Ubuntu"/>
    <addaction name="actionMS_Store_for_OpenSUSE_Tumbleweed"/>
    <addaction name="actionMS_Store_for_Debian"/>
    <addaction name="actionMS_Store_For_WSATools"/>
    <addaction name="actionGo_to_opengapps_page"/>
    <addaction name="actionGo_to_rg_adguard_page_where_wsa_was_downloaded"/>
   </widget>
   <widget class="QMenu" name="menuSwitch_Method">
    <property name="title">
     <string>Change Method</string>
    </property>
    <addaction name="actionOnly_WSA_opengapps"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced_Settings">
    <property name="title">
     <string>Advanced Settings</string>
    </property>
    <addaction name="actionshow_commands_to_type"/>
   </widget>
   <addaction name="menuDownload"/>
   <addaction name="menuSwitch_Method"/>
   <addaction name="menuAbout"/>
   <addaction name="menuAdvanced_Settings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMS_Store_for_powershell">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/powershellicon.png</normaloff>:/image/powershellicon.png</iconset>
   </property>
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
  <action name="actionMy_GitHub">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/github.png</normaloff>:/image/github.png</iconset>
   </property>
   <property name="text">
    <string>My GitHub</string>
   </property>
  </action>
  <action name="actionProject_Page">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/github.png</normaloff>:/image/github.png</iconset>
   </property>
   <property name="text">
    <string>Project Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGo_to_licence">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/licence.png</normaloff>:/image/licence.png</iconset>
   </property>
   <property name="text">
    <string>Go to licence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionMS_Store_for_WSL">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/linux.png</normaloff>:/image/linux.png</iconset>
   </property>
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
  <action name="actionMS_Store_for_Ubuntu">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/ubuntu.png</normaloff>:/image/ubuntu.png</iconset>
   </property>
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
  <action name="actionMS_Store_for_OpenSUSE_Tumbleweed">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/opensuse.png</normaloff>:/image/opensuse.png</iconset>
   </property>
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
  <action name="actionMS_Store_for_Debian">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/debian.png</normaloff>:/image/debian.png</iconset>
   </property>
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
  <action name="actionOnly_WSA_opengapps">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/change.png</normaloff>:/image/change.png</iconset>
   </property>
   <property name="text">
    <string>WSA + OpenGapps</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionApplication_About">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/qt.png</normaloff>:/image/qt.png</iconset>
   </property>
   <property name="text">
    <string>QT</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/donate.png</normaloff>:/image/donate.png</iconset>
   </property>
   <property name="text">
    <string>Donate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/info.jpg</normaloff>:/image/info.jpg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionshow_commands_to_type">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show commands to type</string>
   </property>
  </action>
  <action name="actionGo_to_rg_adguard_page_where_wsa_was_downloaded">
   <property name="text">
    <string>Go to rg-adguard page (where wsa was downloaded)</string>
   </property>
  </action>
  <action name="actionGo_to_opengapps_page">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/opengapps.png</normaloff>:/image/opengapps.png</iconset>
   </property>
   <property name="text">
    <string>Go to opengapps page</string>
   </property>
  </action>
  <action name="actionMS_Store_For_WSATools">
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
  <action name="actionMS_Store_for_1">
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
  <action name="actionMS_Store_for_WSA">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/wsa.png</normaloff>:/image/wsa.png</iconset>
   </property>
   <property name="text">
    <string>MS Store for %1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
